.prettierignore

```diff
@@ -1,3 +1,4 @@
+.changeset/
```

Comment:
There's some bug I think with Prettier and changesets not finding our local Prettier config. https://github.com/changesets/changesets/issues/1283#issuecomment-1981708259